src/Project/Project.router.ts

```diff
@@ -34,6 +39,7 @@ const FILE_NAMES = [
 const MIME_TYPES = ['image/png', 'image/jpeg']
 
 const validator = getValidator()
+const scenePreviewMain = fs.readFileSync('src/Project/scene-preview.js')
```

Comment:
should we put the `scene-prevew` in a separate folder? to me it feels a bit off for this .js file to be here in the `Project` directory.

Maybe instead of `src/Project/scene-preview.js` place it on  `static/scene-preview.js` or something like that?